README.md

```diff
@@ -338,7 +337,7 @@ final Impression impression2 = Impression('citrus', [product1]);
 CommerceEvent commerceEvent = CommerceEvent.withImpression(impression1)
   ..impressions.add(impression2)
   ..currency = 'US'
-  ..screenName = 'One Click Purchase';
+  ..screenName = 'ImpressionScren';
```

Comment:
is this a typo?

Reply:
Yes, it should say `ImpressionScreen` . Fixed.
```suggestion
  ..screenName = 'ImpressionScreen';
```